src/views/Validator/ValidatorContent.tsx

```diff
@@ -48,12 +46,12 @@ export function ValidatorContent({
     <ValidatorLayout
       isRunning={isRunning}
       script={script}
-      logs={logs}
-      checks={checks}
+      logs={session?.logs ?? []}
+      checks={session?.checks ?? []}
       details={details}
     >
       <RequestsSection
-        proxyData={proxyData}
+        proxyData={session?.requests ?? []}
```

Comment:
In a future PR, this view will be displayed conditionally depending on whether or not a debugging session has been started. At that point, the coalescing expressions won't be needed.